stringutil/stringutil.go

```diff
@@ -13,6 +13,11 @@ import (
 	"github.com/beaconsoftwarellc/gadget/log"
 )
 
+const (
+	Left int = 1 + iota
+	Right
+)
```

Comment:
```suggestion
type Direction int

const (
	Left Direction = iota
	Right
)
```

Reply:
then update the function signature to take Direction instead of int to clue user's into it's existence.

Reply:
these will need doc strings as well

Reply:
Update pushed. Thanks!